service/src/app/client.py

```
@@ -0,0 +1,116 @@
+from app.config import Config
+from app.database import init_db
+from app.models import Person
+import sqlalchemy
+import dash_ag_grid as dag
+from dash import Dash, html, dcc, callback, Output, Input, State
+import polars as pl
+import plotly.express as px
+from sqlalchemy import Engine
+
+
+def new_app_state() -> AppState:
```

Ruff (0.15.7)

[error] Undefined name `AppState`

(F821)